cmd/podman/stats.go

```diff
@@ -248,13 +253,13 @@ func pidsToString(pid uint64) string {
 
 func getStatsOutputParams(stats *libpod.ContainerStats) statsOutputParams {
 	return statsOutputParams{
-		Container: stats.ContainerID[:12],
-		ID:        stats.ContainerID,
-		CPUPerc:   floatToPercentString(stats.CPU),
-		MemUsage:  combineHumanValues(stats.MemUsage, stats.MemLimit),
-		MemPerc:   floatToPercentString(stats.MemPerc),
-		NetIO:     combineHumanValues(stats.NetInput, stats.NetOutput),
-		BlockIO:   combineHumanValues(stats.BlockInput, stats.BlockOutput),
-		PIDS:      pidsToString(stats.PIDs),
+		Name:     stats.Name,
+		ID:       stats.ContainerID[:idTruncLength],
```

Comment:
I know it should never happen, but being Irish that just means it's going to happen sooner.  Can we do a len check on stats.ContainerID before this line?  In another review a thought had gone up about making a 'ShortID()" utility function somewhere to safely get a short id.  If that hasn't happened, could you make it so?

Reply:
Added shortID() function.